web/src/components/TraceDrawer/index.css

```diff
@@ -0,0 +1,4 @@
+.ant-drawer-body {
```

Comment:
I think you can achieve the same behavior if you add this to the styled component for the resizable drawer by doing something like:

```
const CustomDrawer = styled(Drawer)`
 .ant-drawer-body {
    display: flex;
    flex-direction: column;
}
`
```